scipy/optimize/_differentialevolution.py

```diff
@@ -294,6 +295,53 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
     iteration, rather than multiple times for all the population members; the
     interpreter overhead is reduced.
 
+    An outline of the strategies and their stengths and weaknesses is given by 
+    Qiang and Mitchell (2014) [8]_
+
+    ..math::
+            DE/rand/1 : v_i = x_{r_1} + F_{xc}(x_{r_2} − x_{r_3})
+
+            DE/rand/2 : v_i = x_{r_1} + F_{xc}(x_{r_2} − x_{r_3}) 
+                                    + F_{xc}(x_{r_4} − x_{r_5})
+
+            DE/best/1 : v_i = x_b + F_{xc}(x_{r_1} − x_{r_2})
+
+            DE/best/2 : v_i = x_b + F_{xc}(x_{r_1} − x_{r_2})
+                                    + F_{xc}(x_{r_3 − x_{r_4})
+
+            DE/current-to-best/1 : v_i = x_i + F_{cr}(x_b − x_i)
```

Comment:
The `strategy` str is `'currenttobest*'`.  i.e. the `DE/current-to-best/1` terminology isn't a 1:1 correspondence with a value you can actually use for `strategy`. Should we use `DE/currenttobest/1` instead?

The same goes for all equations here.

Reply:
I have edited all instances from `DE/current-to-best/1` to `currenttobest1*`